usertest/src/main.rs

```diff
@@ -174,44 +174,6 @@ fn test_write() {
     println!(" OK");
 }
 
-fn test_futex() {
```

Comment:
Any reason we've removed these tests?  I quite like having primitive tests that call the syscall.  I have another PR cooking which adds some edge-case tests too for things like a bogus uaddr, ensuring the fault handling mechanism works correctly, etc.

Reply:
Sure, I've added it back in.